frontend/Interfaces/buddy/DIP/imgcodecs/loadsave.hpp

```diff
@@ -0,0 +1,187 @@
+#ifndef _LOADSAVE_HPP_

```

Comment:
Add opencv's license and add a comment stating that the file was modified by us based on the xxx file of opencv.